src/__tests__/unit/calculate.test.js

```diff
@@ -0,0 +1,26 @@
+import calculate from '../../logic/calculate';
+
+describe('Validate calculate functionality', () => {
+  const num = { total: null, next: null, operation: null };
+  test('multiplication', () => {
+    let result = calculate(num, '10');
+    result = calculate(result, 'x');
+    result = calculate(result, '5');
+    result = calculate(result, '=');
+    expect(result.total).toBe('50');
```

Comment:
- [ ] Please use the React Testing library to stimulate user interaction on the calculator.

Reply:
Hello, @mikenath223 , What does this mean in context? We assumed did so. However, Encase we're missing a point, kindly do share more insight on what you mean.

Cheers 🥂

Reply:
Actioned. ☑️